.travis.yml

```diff
@@ -18,20 +18,25 @@ addons:
         packages:
             - moreutils
             - gettext
+    # seems no effect #         - php5-common
```

Comment:
php * versions are already installed in travis build images

Reply:
oops missing to clen, will amend